domain/removal/state/model/secret_test.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package model
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/juju/tc"
+
+	"github.com/juju/juju/core/network"
+	loggertesting "github.com/juju/juju/internal/logger/testing"
+)
+
+type secretSuite struct {
+	baseSuite
+}
+
+func TestSecretSuite(t *testing.T) {
+	tc.Run(t, &secretSuite{})
+}
+
+func (s *secretSuite) TestDeleteApplicationOwnedSecretContent(c *tc.C) {
+	st := NewState(s.TxnRunnerFactory(), loggertesting.WrapCheckLog(c))
+
+	ctx := c.Context()
+	sec := s.addSecretRevisionsAndContent(c)
+	app, _ := s.addAppAndUnit(c)
```

Comment:
Elsewhere we use `uri` and `appUUID` or similar names. Did we want to be consistent?